### PR TITLE
Adding project "PR Metrics"

### PR DESCRIPTION
**Pull request by muiriswoulfe**
Adding "PR Metrics" to the list of projects, as this project has been very successful with customers and this website should help further this success and bring more input from the OSS community.

### CODE REVIEW

**Comment by jeffwilcox** [comment]
@emmairwin @fossygirl this update points to a marketplace URL, instead of the open source GitHub repo; do we have a policy or preference for the site?

File: _projects/pr-metrics.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+title: PR Metrics
+logo: pr-metrics.png
+description: An Azure DevOps build task that augments Azure DevOps and GitHub pull request titles, letting reviewers quickly determine PR size and test coverage.
+linkText: Try PR Metrics
+projectUrl: https://marketplace.visualstudio.com/items?itemName=ms-omex.prmetrics
```
Comment (jeffwilcox) [inline _projects/pr-metrics.md:6]: I wonder if instead this should point to https://github.com/microsoft/OMEX-Azure-DevOps-Extensions vs the actual install URL... asking the team for input.

Comment (muiriswoulfe) [reply _projects/pr-metrics.md:6]: I only set it like the Marketplace URL as other projects seemed to be doing similar things – e.g., [AccessibilityInsights](https://github.com/microsoft/opensource.microsoft.com/blob/main/_projects/accessibility-insights.md) or [Blazor](https://github.com/microsoft/opensource.microsoft.com/blob/main/_projects/blazor.md).

But those examples are slightly different, so I'm happy to switch to the GitHub URL if we believe it is preferable. Just let me know.

Comment (emmairwin) [reply _projects/pr-metrics.md:6]: My preference would be for the GitHub repo, the readme can point to Marketplace (etc).

Comment (muiriswoulfe) [reply _projects/pr-metrics.md]: Makes sense, I've updated it now.

**Review by emmairwin — APPROVED** [review]

**Comment by muiriswoulfe** [comment]
@emmairwin @jeffwilcox Looks like the Docker login is failing during the build due to a missing username value. Would you be able to take a look into that? Thanks

**Comment by jeffwilcox** [comment]
> @emmairwin @jeffwilcox Looks like the Docker login is failing during the build due to a missing username value. Would you be able to take a look into that? Thanks

For compliance reasons Actions is not currently used. Seems like we're Ok to merge this, though. Thanks!

**Pull request merged**